web/src/components/pages/Login/component.tsx

```diff
@@ -42,17 +46,20 @@ export const LoginPage = (): JSX.Element => {
     let navigate = useNavigate();
 
     const api = new Api();
+    const dispatch = useDispatch();
 
     const login = async () => {
-        setIsLoading(true);
         try {
-            // TODO save user data to reducer
-            const userData = (await api.post('url..', form)) as User;
+            const userData = (await api.post('/login', form)) as User;
             setLocalStorage(USER_TOKEN, userData.token);
+            dispatch(saveUserDataAction(form));
```

Comment:
we want to store userData that comes from api (check api response). not form. PASSWORD and token cannot be stored here :p